src/core/constants.rs

```diff
@@ -38,4 +38,5 @@ pub const TYPE_STRING_UTF8: [u8; 2] = [0xAC, 0x08];
 pub const TYPE_STRING_UTF16: [u8; 2] = [0x9C, 0x16];
 
 
-pub const SCRIPTSLOG_FILE_NAME: &str = "scriptslog.txt";
\ No newline at end of file
+pub const SCRIPTSLOG_FILE_NAME: &str = "scriptslog.txt";
+pub const LINUX_STEAM_PFX_PATH: &str = ".local/share/Steam/steamapps/compatdata/292030/pfx/drive_c/users/steamuser/Documents"; // it's ok to use forward slashes because this is Linux-specific
```

Comment:
Are you sure this path is universal for all linux steam installations? Maybe a better solution would be to just allow passing a custom path to the scriptslog in the CLI?

Reply:
This is the standard Steam prefix location and I think we could keep this as default, but yeah I think it would be better for people with a custom Steam location or a GOG installation if we allowed user input. Good thinking.

Reply:
Should I close this PR and add the option for custom path then?

Reply:
I think it would be better if you added commits to this PR rather than rewrite it.